bindsnet/network/topology.py

```diff
@@ -623,91 +626,8 @@ def normalize(self) -> None:
         Normalize weights so each target neuron has sum of connection weights equal to ``self.norm``.
         """
         if self.norm is not None:
-            self.w = self.w.view(self.source.n, self.target.n)
-            self.w *= self.norm / self.w.sum(0).view(1, -1)
-            self.w = self.w.view(*self.source.shape, *self.target.shape)
-
-    def reset_(self) -> None:
-        # language=rst
-        """
-        Contains resetting logic for the connection.
-        """
-        super().reset_()
-
-
-class MeanFieldConnection(AbstractConnection):
-    # language=rst
-    """
-    A connection between one or two populations of neurons which computes a summary of the pre-synaptic population to
-    use as weighted input to the post-synaptic population.
-    """
-
-    def __init__(
-        self,
-        source: Nodes,
-        target: Nodes,
-        nu: Optional[Union[float, Sequence[float]]] = None,
-        weight_decay: float = 0.0,
-        **kwargs
-    ) -> None:
-        # language=rst
-        """
-        Instantiates a :code:`MeanFieldConnection` object.
-
```

Comment:
Do we not need MeanFieldConnection anymore?

Reply:
We should retain MeanFieldConnection? The subject going to do a comeback

Reply:
I don't think we're ever going to use it, but if we do, we can always put it back in (it will always exist in the git history). I don't know if the implementation is correct.

Reply:
If we need it, we can add it back in. I don't think the implementation is correct, though, so it makes sense to remove it for now. It will still live in the git history, so we can get it back any time.

Reply:
I'm using it currently now for Ray/Robert CAN program.
If its not correct we should fix it.

Reply:
Okay, I'll add it back in!